### PR TITLE
Issue #14084: fix Checker violation for nullable return

### PR DESCRIPTION
**Pull request by romani**
Issue #14084

https://stackoverflow.com/a/35875417

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java [hunk 1]
```diff
@@ -307,7 +309,7 @@ private static DetailAST getBraceFromSwitchMember(DetailAST ast) {
      * @return {@code DetailAST} if the first child is {@code TokenTypes.SLIST},
      *     {@code null} otherwise.
      */
-    private static DetailAST getBraceAsFirstChild(DetailAST ast) {
+    private static @Nullable DetailAST getBraceAsFirstChild(DetailAST ast) {
```
Comment (nrmancuso) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java:312]: Please make annotation above method definition. We should also make a check module for this. As we grow in annotation usage, we will not want to inline all of them.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

**Review by nrmancuso — APPROVED** [review]
Awesome!

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java (hunk 1 above)
Comment (nrmancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java]: https://github.com/checkstyle/checkstyle/issues/14101#issue-2026222144

Comment (nrmancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java]: @romani assigned you for approval ^

Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java]: approved, I surprised that it was not done from the beginning.

File: config/import-control.xml [hunk 2]
```diff
@@ -20,6 +20,7 @@
   <allow pkg="java.net"/>
   <allow pkg="java.util"/>
   <allow pkg="javax.xml.parsers"/>
+  <allow class="javax.annotation.Nullable"/>
```
Comment (rnveach) [inline config/import-control.xml:23]: Move up one line so it is alphabetical.

**Review by rnveach — APPROVED** [review]

File: config/import-control.xml (hunk 2 above)
Comment (romani) [reply config/import-control.xml:22]: done

**Review by rnveach — APPROVED** [review]

**Pull request merged**